### PR TITLE
Two doors

### PR DESCRIPTION
**Pull request by gyes51y767p**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,65 @@
...
+* Use the scirpt for esp32 form [here](door_sensor/door_sensor.ino)
+* Open the script in Arduino IDE and edit the personal password and subscribe/publish topic
+* Upload the script to ESP32
+* Connect the ESP32 to the magnetic switch and power supply
+* Done!
+
+For Raspberry Pi:
+* Use the script for Raspberry Pi from [here](musicbox/musicbox.py)
+* Open terminal to musicbox folder
+* Create a python 3.9 virtual environment
```
Comment (jeffwrule) [inline README.md:44]: I recommend Python 3.9 or higher here.  There is no expectation that it won't work with future versions.  We need a version of Python that is compatible with the paho.mqtt library.

File: door_sensor/door_sensor.ino [hunk 2]
```diff
@@ -12,23 +12,23 @@
 
 // wifi config
 const char ssid[] = "LAK-FOXSPARROW";		// wifi ssid
```
Comment (jeffwrule) [inline door_sensor/door_sensor.ino:14]: change to <your-wifi-zone> in the quotes

File: door_sensor/door_sensor.ino [hunk 3]
```diff
@@ -12,23 +12,23 @@
...
-const char pass[] = "4793263208";				// wifi password
+const char pass[] = "";				// wifi password
```
Comment (jeffwrule) [inline door_sensor/door_sensor.ino:15]: change to <your-wifi-password>

File: door_sensor/door_sensor.ino [hunk 4]
```diff
@@ -12,23 +12,23 @@
...
 
 // mqt config
 const char mqtt_broker_address[] = "61850993d7e340d7b170bcd12078ef88.s2.eu.hivemq.cloud";
```
Comment (jeffwrule) [inline door_sensor/door_sensor.ino:18]: change to <mqtt-broker-host>

File: door_sensor/door_sensor.ino [hunk 5]
```diff
@@ -12,23 +12,23 @@
...
 const int mqtt_port = 8883;
 const char mqtt_user[] = "doorman";
```
Comment (jeffwrule) [inline door_sensor/door_sensor.ino:20]: change to mqtt-broker-user

File: door_sensor/door_sensor.ino [hunk 6]
```diff
@@ -12,23 +12,23 @@
...
 const char mqtt_password[] = "";
```
Comment (jeffwrule) [inline door_sensor/door_sensor.ino:21]: change to <mqtt-broker-password>

File: musicbox/working_process_images/direct_voltage4.jpg
Comment (jeffwrule) [inline musicbox/working_process_images/direct_voltage4.jpg:1]: delete these we did not use the power adaptor board.

**Review by jeffwrule — CHANGES_REQUESTED** [review]
I think you should move the passwords/hosts configs into a musicbox.ini file (a typical thing for editable parameters).   The configs for the music files and timing should remain in the code.   This is pretty simple to do and makes means you can run the code with different servers w/o having edit the python file.

File: musicbox/working_process_images/direct_voltage4.jpg
Comment (gyes51y767p) [reply musicbox/working_process_images/direct_voltage4.jpg:1]: done

File: door_sensor/door_sensor.ino (hunk 6 above)
Comment (gyes51y767p) [reply door_sensor/door_sensor.ino:21]: done

File: door_sensor/door_sensor.ino (hunk 5 above)
Comment (gyes51y767p) [reply door_sensor/door_sensor.ino]: done

File: README.md (hunk 1 above)
Comment (gyes51y767p) [reply README.md:48]: done

**Review by jeffwrule — APPROVED** [review]
These changes look good.

**Pull request merged**